Alexey/Text/pig_latin/test/test_pig_latin.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from Alexey.Text.pig_latin.main.pig_latin import translater
+
+
+class TestPigLatin(unittest.TestCase):
+
+    def test_send_none_and_expect_attribute_error(self):
+        with self.assertRaises(AttributeError) as context:
```

Comment:
You have checked exception twice here. Please remove one check

Reply:
done

Reply:
Nope, wrote in pm